packages/normalize/css/index.css

```diff
@@ -9,6 +9,13 @@
   font-display: swap;
   font-weight: 100 900;
```

Comment:
Would be great if there was a "font-style range" option like there is for font-weight.